x/claims/keeper/abci_test.go

```diff
@@ -139,15 +159,30 @@ func (suite *KeeperTestSuite) TestClawbackEmptyAccounts() {
 				suite.app.ClaimsKeeper.SetClaimsRecord(suite.ctx, addr, types.ClaimsRecord{})
 			},
 		},
+		{
+			"balance more than dust, is ignored",
```

Comment:
Add case with only IBC coins and no claim denomination coins

Reply:
That is tested here - https://github.com/evmos/evmos/blob/clawback-issue/x/claims/keeper/abci_test.go#L160